spaceship/spec/tunes/application_spec.rb

```diff
@@ -139,30 +149,30 @@
         mock_client_response(:get_builds_for_train, with: hash_including(train_version: '1.0')) do
           [
             {
-              id: 1,
-              appAdamId: 10,
-              trainVersion: '1.0',
-              uploadDate: '2017-01-01T12:00:00.000+0000',
-              externalState: 'testflight.build.state.export.compliance.missing'
+                id: 1,
```

Comment:
why these formatting changes?

Reply:
rubocop induced is my guess. I ran rubocop while uploading a patch and it was still kept as is. I can reverse it, if you like @janpio